include/mbedtls/ssl.h

```diff
@@ -110,7 +110,8 @@
 /* Error space gap */
 /* Error space gap */
 /* Error space gap */
-/* Error space gap */
+/** Cache entry not found */
+#define MBEDTLS_ERR_SSL_CACHE_NOT_FOUND                   -0x7E80
```

Comment:
It looks macro for `ssl` error is defined in descending order. `MBEDTLS_ERR_SSL_CACHE_NOT_FOUND` should be moved after `MBEDTLS_ERR_SSL_HW_ACCEL_FAILED`

Reply:
There is no error code space after `0x7F80`. Please check [include/mbedtls/error.h](https://github.com/Mbed-TLS/mbedtls/blob/development/include/mbedtls/error.h)

Reply:
Oh right, I was concentrating on the line after `MBEDTLS_ERR_SSL_ALLOC_FAILED` but didn't notice the line before `MBEDTLS_ERR_SSL_CACHE_NOT_FOUND`. But it feels a little bit strange to me that before and after line 118, macro is defined in different ascending and descending order.

Reply:
It's not such a big problem. The ordering has been there for nearly [ten years](https://github.com/lpy4105/mbedtls/commit/05ef835b6a97ca85c331c0160278dabd6c92d99b).  Since the error code assignment is complicated now, I believe there are two extension directiions when two modules are sharing the same space.